style.css

```diff
@@ -0,0 +1,186 @@
+
```

Comment:
Try an implement styles for mobile and tablet, so that it works across all devices. https://www.uxpin.com/studio/blog/a-hands-on-guide-to-mobile-first-design/ https://www.w3schools.com/css/css_rwd_mediaqueries.asp